src/server/game/Entities/Creature/GossipDef.cpp

```diff
@@ -400,110 +401,42 @@ void PlayerMenu::SendQuestGiverStatus(uint8 questStatus, ObjectGuid npcGUID) con
 
 void PlayerMenu::SendQuestGiverQuestDetails(Quest const* quest, ObjectGuid npcGUID, bool activateAccept) const
 {
-    std::string questTitle            = quest->GetTitle();
-    std::string questDetails          = quest->GetDetails();
-    std::string questObjectives       = quest->GetObjectives();
-    std::string questAreaDescription  = quest->GetAreaDescription();
+    WorldPackets::Quest::QuestGiverQuestDetails packet;
+
+    packet.Title = quest->GetTitle();
+    packet.Details = quest->GetDetails();
+    packet.Objectives = quest->GetObjectives();
 
     LocaleConstant localeConstant = _session->GetSessionDbLocaleIndex();
     if (localeConstant != LOCALE_enUS)
     {
         if (QuestLocale const* localeData = sObjectMgr->GetQuestLocale(quest->GetQuestId()))
         {
-            ObjectMgr::GetLocaleString(localeData->Title,           localeConstant, questTitle);
-            ObjectMgr::GetLocaleString(localeData->Details,         localeConstant, questDetails);
-            ObjectMgr::GetLocaleString(localeData->Objectives,      localeConstant, questObjectives);
-            ObjectMgr::GetLocaleString(localeData->AreaDescription, localeConstant, questAreaDescription);
```

Comment:
Why this was removed?

Reply:
Because `questAreaDescription` is not used for this packet (see old code, its not written anywhere)